lib/Sema/CodeSynthesisDistributedActor.cpp

```diff
@@ -1066,3 +1065,30 @@ bool CanSynthesizeDistributedActorCodableConformanceRequest::evaluate(
          TypeChecker::conformsToKnownProtocol(
              idTy, KnownProtocolKind::Encodable, actor->getParentModule());
 }
+
+NormalProtocolConformance *
+GetDistributedActorAsActorConformanceRequest::evaluate(
+    Evaluator &evaluator, ProtocolDecl *distributedActorProto) const {
+  auto &ctx = distributedActorProto->getASTContext();
+  auto swiftModule = ctx.getStdlibModule();
+
+  auto actorProto = ctx.getProtocol(KnownProtocolKind::Actor);
+
+  auto ext = findDistributedActorAsActorExtension(
+      distributedActorProto, swiftModule);
+  if (!ext)
+    return nullptr;
+
+  auto genericParam = GenericTypeParamType::get(/*isParameterPack=*/false,
+                                                /*depth=*/0, /*index=*/0, ctx);
```

Comment:
This seems to be fine...? I guess so, at least real project also is happy with just inventing a generic type here. If there's better way let me know please @slavapestov @xedin

Reply:
This looks fine to me

Reply:
thanks!